packages/query/src/block-processor.ts

```diff
@@ -373,8 +373,7 @@ export class BlockProcessor implements BlockProcessorInterface {
       );
     }
 
-    const isLastBlockOfEpoch = !!compactBlock.epochRoot;
-    if (isLastBlockOfEpoch) {
+    if (compactBlock.epochRoot) {
```

Comment:
issue: we lost a bit of documentation here that informed the dev that the presence of this field means it's the last block of the epoch. Perhaps a code comment can bring that info back?

Reply:
added comment